src/legacy/views/preview/contact-preview-content.jsx

```diff
@@ -339,12 +337,23 @@ function ContactPreviewContent({ contact, onEdit, onDelete, onMail, onMove }) {
 		() => tagsFromStore[contact?.tags?.[0]]?.name,
 		[contact?.tags, tagsFromStore]
 	);
+
+	const openURLInNewTab = (url) => {
+		// Add the protocol if it's missing
+		let urlWithProtocol = url;
+		if (!urlWithProtocol.startsWith('http://') && !urlWithProtocol.startsWith('https://')) {
+			urlWithProtocol = `https://${urlWithProtocol}`;
```

Comment:
I'm not sure about this assumption. IMO we cannot be sure about which protocol is actually used. If the https protocol is not active on the destination server the link will result broken and it will appear as an error from Carbonio

Reply:
Giving a deeper look it seems that prepending a protocol is one of the few way to avoid the browser to consider the protocol-less URL as a relative one.
Another solution can be to prepend a // to transform the URL into a protocol-relative URL (see https://en.wikipedia.org/wiki/URL#prurl), but this will result in the browser _composing_ the final URL using the current protocol which can be not available on the destination server.
In the end it looks there is no a 100% clean solution to handle this scenario (which is caused by incomplete data provided by the user): prepending a protocol seems to be the best approximation possible and since the browsers are now defaulting to HTTPS we can follow the same policy